dockerfiles/run/guice/cassandra-ldap/destination/conf/lmtpserver.xml

```diff
@@ -18,6 +18,8 @@
   under the License.                                           
  -->
 
+<!-- Read https://james.apache.org/server/config-lmtp.html for further details -->
```

Comment:
dead link?

Reply:
more https://james.apache.org/server/config-smtp-lmtp.html#LMTP_Configuration

Reply:
Yep better with the tag indeed (same goes for smtp)